pipeline-runner/init.R

```diff
@@ -282,6 +283,43 @@ call_data_processing <- function(task_name, input, pipeline_config) {
     return(message_id)
 }
 
+#
+# start_heartbeat(task_token, aws_config)
+# IN task_token, aws_config
+#
+# Sends a heartbeat to the state machine every 10 seconds
+# Once the task is completed the heartbeat will fail accordingly with a
+# task timeout and exit the loop and a new heartbeat will be set up by next task.
```

Comment:
The task timeout is 10800 seconds (3 hours). Does this mean that this process will run for 3 hours before failing?

Reply:
no:
1. as soon as the task is completed the heartbeat fails and the loop stops
2. when a pipeline finishes the pod and everything running inside is just killed